docs/book/user-guide/component-guide/model-deployers/seldon.md

```diff
@@ -153,50 +153,50 @@ the [`rclone` documentation for various providers](https://rclone.org/).
 Example of configuring a Seldon Core secret for AWS S3:
 
 ```shell
-zenml secret create s3-seldon-secret \
---rclone_config_s3_type="s3" \ # set to 's3' for S3 storage.
---rclone_config_s3_provider="aws" \ # the S3 provider (e.g. aws, Ceph, Minio).
---rclone_config_s3_env_auth=False \ # set to true to use implicit AWS authentication from EC2/ECS meta data
+zenml secret create s3-seldon-secret --values= \
```

Comment:
This is one example where storing the values as a YAML file and passing it using the `--values=@path/to/file.yaml` would be preferable to writing this whole complicated, multi-line string in the command line.

You could try the same exercise as before here: ask yourself what you would choose to do here to make your life easier:

* use multiple `--key=value` arguments
* use the inline `--values=<JSON>` option
* use the `--values=@path/to/file.yaml` option

Putting yourself in the user's shoes is the best way to understand how your work will impact everyone else.